marklogic-data-hub-central/ui/src/components/header/system-info.tsx

```diff
@@ -156,7 +156,7 @@ const SystemInfo = (props) => {
               <Col>
                 <Card size="small" className={styles.download} >
                   <div className={styles.title}>Download Hub Central Files</div>
-                  <p>Download a zip containing files for everything that Hub Central supports creating and updating. Does not include project files that Hub Central does not allow creating or updating, nor does it contain Data Hub Framework project files. Can be used to update an existing local project with updates made in Hub Central.</p>
+                  <p>Download updates made to the project via Hub Central so that they can be applied to an existing local project. The updates include the files Hub Central supports creating and updating only.</p>
```

Comment:
I think it would be clearer if we could rewrite the last line as
`
The updates include only those files that can be created and updated via Hub Central.
`